test/powershell/Modules/Microsoft.PowerShell.Management/Get-Item.Tests.ps1

```diff
@@ -141,5 +141,24 @@ Describe "Get-Item" -Tags "CI" {
 
             (get-item env:\testvar).Value | Should -BeExactly $expectedValue
         }
+
+        It "get-item reports the effective value among accidental case-variant duplicates on Windows" -skip:(-not $isWindows) {
+            if (-not (Get-Command -ErrorAction Ignore node.exe)) {
+                Write-Warning "Test skipped, because Node.js is required to run it."
```

Comment:
We should not depend on Node.js - please implement a mock/help function.

Reply:
@iSazonov: Node.js is the only way that I know of to create this scenario - all Windows APIs seemingly - and sensibly - prevent the creation of case-variant duplicates.

Reply:
So, unless we do come up with a third-party-free way to mock this up, we're faced with the choice between leaving it in with the warning or removing the test altogether.

Reply:
I wonder - what api Node.js use on _Windows_?

Reply:
https://stackoverflow.com/questions/573817/where-are-environment-variables-stored-in-registry

Reply:
@iSazonov: Re what API Node.js uses - that's an excellent question, but, honestly, not one I feel personally inspired to investigate. As for the SO link: The Windows registry is inherently case-INSENSITIVE, so by definition it cannot be involved here - we're talking about dynamically created process-level environment variables.

Reply:
@iSazonov It's definitely not the registry as it is updating the process environment, which is in-memory.

Reply:
I agree the node requirement should not be added to a CI test.  Can you move this to a different describe and make it a Feature test?

Reply:
@TravisEz13: Happy to do it, but I need guidance: you mean a separate `Describe` block with `-tags 'Scenario'`? In the same file? Anything else (more formal description of the context and what's needed to run the test)?

Reply:
@yes, the implementation details you describe look correct.  The goal is just to prevent everyone from having to have Node.js.  We can easily enough ensure it's available on the machine we run the scenario tests.